picard/coverart/image.py

```diff
@@ -307,7 +310,7 @@ def set_data(self, data):
 
         try:
             (self.width, self.height, self.mimetype, self.extension,
-             self.datalength) = imageinfo.identify(data)
+             self.datalength) = astuple(imageinfo.identify(data))
```

Comment:
Instead of using tuples we could just use attributes:

```python
   info = imageinfo.identify(data)
   self.width, self.height = info.width, info.height
   self.mimetype = info.mime
   self.extension = info.extension
   self.datalength = info.datalen
```

I know it's longer but experience shows the code is easier to manage if we don't have too much positional arguments and unnamed tuples.